_includes/journals/volume1/issue1/paper1.html

```diff
@@ -0,0 +1,30 @@
+
+<h1>{{page.title}}</h1>
+
+<strong>Author(s): {{page.author}}</strong> <br>
+<strong><a target="_blank" href="{{paper.link}}">Get PDF</a></strong>
+
+<div class="article-parents col-md-12">
+    <div class="row">
+        <ul class="small list-inline col-md-12 article-journal-name" style="vertical-align:top">
+            <li style="margin: 0;"><strong style="font-size: larger;">Journal of LatinX in AI (LXAI)</strong></li>
+            <li>{{page.grand_parent}},</li>
```

Comment:
Rather than grand_parent or parent as variable names, let's be more specific, should be volume and issue.